src/app/navbar/navbar-section/navbar-section.component.scss

```diff
@@ -1,5 +1,5 @@
-.navbar-section {
-  display: flex;
-  align-items: center;
-  height: 100%;
+:host {
+  li.ds-menu-item-wrapper {
+
+  }
```

Comment:
Looks like this can be removed

Reply:
I left it to make clear that this is the correct way to customise the items. Should I remove it anyway or should I add a comment to explain that?